cli/tests/pcluster/models/cluster_dummy_model.py

```diff
@@ -66,10 +71,10 @@ def dummy_head_node():
     return HeadNode(instance_type="fake", networking=head_node_networking, ssh=ssh, dcv=head_node_dcv)
 
 
-def dummy_cluster():
+def dummy_cluster(mocker):
```

Comment:
let's add in the dummy_cluster a raid volume.

Reply:
Done